Sources/PackageModel/PackageModel.swift

```diff
@@ -126,7 +126,7 @@ extension Manifest {
         return self.toolsVersion >= .v5_8
     }
     public var usePackageNameFlag: Bool {
-        return self.toolsVersion >= .vNext
+        return self.toolsVersion >= .v5_9
```

Comment:
Since SE-0386 is still under review, we shouldn't flip this yet. So either we can leave this PR open until the proposal is accepted or you could leave this at `vNext` to merge now and do a tiny follow-up to flip it later.

Reply:
ok I'll revert back to vNext for this PR and flip it later